lib/execution_engine2/db/models/models.py

```diff
@@ -336,7 +336,11 @@ class Job(Document):
         default=False
     )  # Marked true when all retry steps have completed
 
-    meta = {"collection": "ee2_jobs"}
+    # See https://docs.mongoengine.org/guide/defining-documents.html#indexes
+    # Hmm, are these indexes need to be + or - ?
+    indexes = [("status", "batch_job"), ("status", "queued")]
```

Comment:
What's the query for the status/batch_job index?

Reply:
https://github.com/kbase/execution_engine2/blob/develop/bin/PurgeBadJobs.py#L103